build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/GoDriver.java

```diff
@@ -29,13 +31,55 @@ public class GoDriver implements Serializable {
     private final CommandExecutor commandExecutor;
     private final File workingDirectory;
     private final Log logger;
+    private static final String defaultExecutablePath = "go";
```

Comment:
```suggestion
    private static final String DEFAULT_EXECUTABLE_NAME = "go";
```

Let's also put it above, near the other private static final variables.